editor/js/libs/ui.js

```diff
@@ -124,6 +124,12 @@ UIElement.prototype = {
 
 		return this;
 
+	},
+
+	getIndexOfChild: function ( element ) {
+
+		return Array.prototype.indexOf.call( this.dom.children, element.dom );
```

Comment:
see https://stackoverflow.com/questions/222841/most-efficient-way-to-convert-an-htmlcollection-to-an-array